test/functional/wallet_descriptor.py

```
@@ -21,6 +30,42 @@ def set_test_params(self):
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
         self.skip_if_no_sqlite()
+        self.skip_if_no_py_sqlite3()
+
+    def test_concurrent_writes(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_concurrent_writes`

Add return type annotation: `None`

(ANN201)